libs/gost_engine/Makefile

```diff
@@ -0,0 +1,79 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gost_engine
+PKG_VERSION:=1.1.0.3
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/gost-engine/engine/archive/v$(PKG_VERSION)
+PKG_HASH:=fff725052e82c9adb5b738729b30141f61ac91fa457a4f4b5de18b8b24092f75
+
+PKG_MAINTAINER:=Artur Petrov <github@phpchain.ru>
+PKG_LICENSE:=OpenSSL
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C "$(PKG_BUILD_DIR)" --strip-components=1 -xzf "$(DL_DIR)/$(PKG_SOURCE)"
```

Comment:
Instead of this you can use PKG_BUILD_DIR

Reply:
This is on me.  I should have let @6ec123321 keep my comments in the Makefile. 🤦‍♂️ 
The tarball installs to just `engine-$(PKG_VERSION)`, which is too generic, don't you  think?  If you know a better way to handle it, please show me.

Reply:
Does it conflict with anything?

Reply:
Not really.  You have to override one default (`PKG_UNPACK`) or another (`PKG_BUILD_DIR`).  Overriding PKG_UNPACK makes it a bit easier to find the directory by hand.  [I had doubts myself on which one would be best](https://github.com/openwrt/packages/pull/14765#issuecomment-780921453), so won't insist any longer on this.

Reply:
Meh OK.